test/services/return-requirements/submit-add-note.service.test.js

```diff
@@ -0,0 +1,149 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const SessionHelper = require('../../support/helpers/session.helper.js')
+
+// Thing under test
+const SubmitAddNoteService = require('../../../app/services/return-requirements/submit-add-note.service.js')
+
+describe('Submit Add Note service', () => {
+  let payload
+  let session
+  const user = { username: 'carol.shaw@atari.com' }
+
+  beforeEach(async () => {
+    await DatabaseSupport.clean()
+
+    session = await SessionHelper.add({
+      data: {
+        checkAnswersVisited: true,
+        licence: {
+          id: '8b7f78ba-f3ad-4cb6-a058-78abc4d1383d',
+          currentVersionStartDate: '2023-01-01T00:00:00.000Z',
+          endDate: null,
+          licenceRef: '01/ABC',
+          licenceHolder: 'Turbo Kid',
+          startDate: '2022-04-01T00:00:00.000Z'
+        },
+        journey: 'no-returns-required',
+        note: '',
```

Comment:
I'm fine with either the property not existing or an existing note being on the session as part of the test. But it'd never be `note: ''` anymore so we need to correct it.